lib/go/test/flow_epoch_test.go

```diff
@@ -847,9 +847,21 @@ func TestEpochQCDKG(t *testing.T) {
 
 	t.Run("Can end the Epoch and start a new Epoch", func(t *testing.T) {
 
-		tx := createTxWithTemplateAndAuthorizer(b, templates.GenerateEpochCalculateSetRewardsScript(env), idTableAddress)
+		result := executeScriptAndCheck(t, b, templates.GenerateGetFlowTotalSupplyScript(env), nil)
+		assertEqual(t, CadenceUFix64("7000000000.0"), result)
 
-		_ = tx.AddArgument(CadenceUFix64("1300000.0"))
+		tx := createTxWithTemplateAndAuthorizer(b, templates.GenerateEpochSetAutomaticRewardsScript(env), idTableAddress)
+
+		_ = tx.AddArgument(cadence.NewBool(true))
+
+		signAndSubmit(
+			t, b, tx,
+			[]flow.Address{b.ServiceKey().Address, idTableAddress},
+			[]crypto.Signer{b.ServiceKey().Signer(), IDTableSigner},
+			false,
+		)
+
+		tx = createTxWithTemplateAndAuthorizer(b, templates.GenerateEpochCalculateSetRewardsScript(env), idTableAddress)
```

Comment:
Why is calculating the rewards with an admin transaction needed here? Wouldn't the epoch transition do that automatically, given that we enable automatic reward payouts above?

Reply:
As far as I know, getting view counts in the emulator still isn't possible, so we have been able to test automated epochs yet with the emulator. All the tests still just trigger the transitions on their own

Reply:
Ah right, good point. Forgot about that.